config/initializers/bugsnag.rb

```diff
@@ -1,3 +1,3 @@
 Bugsnag.configure do |config|
-  config.api_key = '4a1ee1b8b01c7b53e301bea9dbe47659'
+  config.api_key = ENV['BUGSNAG_API_KEY']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.